### ISSUE TITLE
Send keys with shift toggle is not working for current firefox

### ISSUE THREAD

**Issue by lread** [clj-commons/etaoin#387]
## Currently 

Etaoin has an ide test that exercises shift key toggling.
https://github.com/igrishaev/etaoin/blob/e8d77afec20f42be87c974ac5a09d1dbc116cb4d/resources/ide/test.side#L362-L365

The idea is this should effectively send `LOGin`. 
This works fine on Ubuntu on CircleCI.

## But...

In local testing I found it to fail on macOS and Windows.

When things work we see log lines that look like this:
```
INFO  etaoin.ide.api: Command: sendKeys on 'id=simple-input' with value: '${KEY_SHIFT}log${KEY_SHIFT}in' / OK
INFO  etaoin.ide.api: Command: assertValue on 'id=simple-input' with value: 'LOGin' / OK
```

But I see failures for firefox that look like this:
```
INFO  etaoin.ide.api: Command: sendKeys on 'id=simple-input' with value: '${KEY_SHIFT}log${KEY_SHIFT}in' / OK
INFO  etaoin.ide.api: Command: assertValue on 'id=simple-input' with value: 'LOGin' / Failed: Assert failed: 
Assert command:"assertValue"
Expected: LOGin
Actual: LOGIN
(= actual-val value)
```

It seems like the 2nd shift is not acting as a toggle.

## So what's up?

I tried experimenting locally with the same Ubuntu docker image we are currently using on CircleCI. This image has the following installed:
- firefox v83.0
- geckodriver v0.28.0

Tests still passed when changing geckodriver to v0.29.0 and then v0.31.0 (the current release).

I then decided to bump firefox to v100.0 (current release) and saw the same failure I witnessed locally on macOS and Windows.

I did a little more testing against various versions of firefox on Ubuntu. It looks like things started to go bad in v99.

- v83.0 released November, 2020 ✅ passed (what we are testing against on CircleCI)
- v90.0.1 released July, 2021 ✅ passed
- v95.0.2 released December, 2021 ✅ passed
- v97.0.2 released February, 2022 ✅ passed
- v98.0.2 released March, 2022 ✅ passed
- v99.0.1 released April, 2022 ❌ failed
- v100.0  released May 3, 2022 ❌ failed (current release)
- v101.0b4 (beta not released yet) ❌ failed 

## So...
I was kind of surprised that this issue seems to be in firefox and not geckodriver.

## What to do?
I'm not sure, this is a pretty recent change in behavior.
One option is to temporarily adapt the test to pass for firefox and document the potential limitation.

**Comment by lread** [comment]
I'm kind of wondering if modifier key toggling is generally supported very well.
Does anybody know? If so, please do share!

**Comment by lread** [comment]
Hmmm... If I look at [the current w3c webdriver spec for send keys](https://w3c.github.io/webdriver/webdriver-spec.html#element-send-keys), I see:

> If the key to type is a [modifier key](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-modifier-key) then set the relevant attribute on [Keyboard Event](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-keyboard-event) to true for the rest of this loop.

So maybe firefox is just trying to adhere to the spec.

**Comment by lread** [comment]
I do see this [bug fix for firefox 99.0](https://bugzilla.mozilla.org/show_bug.cgi?id=1757636) (which is when behaviour changed see above).

I think I'll adapt the test to include only one shift modifier.  
If we do need to support toggling key modifiers to properly support selenium ide scripts, we can raise that as a separate issue.
